src/xml_element.js

```diff
@@ -223,6 +224,14 @@ XMLElement.prototype.populateChildren = function() {
 			}
 		}
 	});
+	// Patch to add "required" attributes. Does require a secondary change to know the "use" attribute also. See xsd2json-use.js : SchemaProcessor.prototype.createDefinition
+	$.each(this.objectType.attributes, function(){
```

Comment:
this is a fairly significant change that wasn't mentioned in the commits.  I'm not opposed to auto-populating required attributes, which a number of people have requested.  But I can't seem to get this to work, do you have any example schema where this is adding in attributes?

Reply:
Was accidentially placed here - was a special use case I didn't plan to bring in here. But I will add a example with an XSD in the demo block.
Will add a switch, to turn this off (which I set to off/false)